ci/tests_all_in_schedule.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+# shellcheck disable=SC1091
+source ci/ci_helpers.sh
+cd src/test/regress
+
+for test_dir in sql spec; do
+    test_names=$(find $test_dir -iname '*.sql' | sed "s#^$test_dir/\\(.*\\)\\.\\(sql\\|spec\\)\$#\\1#g")
```

Comment:
I don't know what happened to my previous comment at this line, so I am rewriting:

I guess here we are getting the basename of each test name. There are 3 different test files, '.sql', '.spec' and '.source', what about we use something simpler:

```bash
find . -iname "*.sql" -exec basename {} .sql \;
```

We can possible have one command for the 3 different type of tests or we can have 3 separate commands. If you use the above command we can also remove the outer loop as well. What do you think?